notebooks/mrc/LLM_reader_predict_mode.ipynb

```diff
@@ -35,25 +46,54 @@
     "passages = [[\"Driving under the influence\\\"\\n \\\"driving under the influence (DUI). However, no matter the level of THC, law enforcement officers base arrests on observed impairment.\\\"\\\" In Colorado, if consumption of marijuana is impairing your ability to drive, \\\"\\\"it is illegal for you to be driving, even if that substance is prescribed [by a doctor] or legally acquired.\\\"\\\" Prescription medications such as opioids and benzodiazepines often cause side effects such as excessive drowsiness, and, in the case of opioids, nausea. Other prescription drugs including antiepileptics and antidepressants are now also believed to have the same effect. In the last ten years, there has been an increase\",\n",
     "              \"Effects of cannabis\\\"\\n \\\"studies have found lower (or no) elevated crash risk estimates\\\"\\\". The study found that \\\"\\\"after adjusting for age, gender, race and alcohol use, drivers who tested positive for marijuana were no more likely to crash than those who had not used any drugs or alcohol prior to driving\\\"\\\". On the other hand, a recent study of Journal of Transport & Health indicated that the numbers of fatal crashes involving marijuana after the recreational marijuana legalization or decriminalization have significantly increased in Colorado, Washington, and Massachusetts. Short-term (one to two hours) effects on the cardiovascular system can include increased heart rate\"\n",
     "              ]]\n",
-    "prompt_prefix = \"Answer the following question using the text\"\n",
-    "prompt_suffix = \"\"\n",
-    "# This creates the following prompt:\n",
```

Comment:
good to keep this comment for gpt-3.5 too, more so because now it is coming as the first example before flan-t5